app/livechat/server/lib/Livechat.js

```diff
@@ -981,13 +981,15 @@ export const Livechat = {
 			return false;
 		}
 
-		const message = `${ data.message }`.replace(/([^>\r\n]?)(\r\n|\n\r|\r|\n)/g, '$1<br>$2');
+		const { message, name, email, department, host } = data;
+		const emailMessage = `${ message }`.replace(/([^>\r\n]?)(\r\n|\n\r|\r|\n)/g, '$1<br>$2');
 
 		const html = `
 			<h1>New livechat message</h1>
-			<p><strong>Visitor name:</strong> ${ data.name }</p>
-			<p><strong>Visitor email:</strong> ${ data.email }</p>
-			<p><strong>Message:</strong><br>${ message }</p>`;
+			<p><strong>Sent from:</strong><a href='${ host }'> ${ host }</a></p>
```

Comment:
Wouldn’t it be nice to check if the host exists?
Because if the data comes from this [method](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/methods/sendOfflineMessage.js#L15), the host link will be blank.

Reply:
Done!